docs/DeveloperGuide.adoc

```diff
@@ -844,13 +882,34 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+=== Use case: Search a person by tag
+*MSS*
+
+1. User requests to search and provides search params
+2. Addr	essBook shows a list of person fitting the given criteria
```

Comment:
Typo here. Also, your product name is no longer AddressBook right?

Reply:
> TODO

This update will be reflected in v1.4